### ISSUE TITLE
Allow passing arguments to bash in Dockerfile

### ISSUE THREAD

**Issue by thesuperzapper** [apache/airflow#8837]
As discussed in  #8579, we should add support for passing arguments to the `bash` command in `entrypoint.sh` of the Dockerfile.

For example:
```{bash}
if [[ ${AIRFLOW_COMMAND} == "bash" ]]; then
   exec "/bin/bash" "${@:2}"
fi
```

**Comment by thesuperzapper** [comment]
@potiuk here is the issue

**Comment by potiuk** [comment]
Fixed in #9162
